service/internal/logger/audit.go

```diff
@@ -0,0 +1,102 @@
+package logger
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type PolicyLog struct {
+	UUID uuid.UUID  `json:"uuid"`
+	Body PolicyBody `json:"body"`
+}
+
+type PolicyBody struct {
+	DataAttributes []SimpleAttribute `json:"dataAttributes"`
+	Dissem         []string          `json:"dissem"`
+}
+
+type SimpleAttribute struct {
+	URI string `json:"attribute"`
```

Comment:
Nit: FQN (fully qualified name) seems like it would be better here for consistency

Reply:
I was mostly basing this off of [how it was implemented in Python](https://github.com/opentdf/backend/blob/dbfcea7594fa72dd49d02a19aa01b36f3a32b7e6/containers/kas/kas_app/tdf3_kas_app/plugins/audit_hooks.py#L31-L77) as well as [the current policy struct](https://github.com/opentdf/platform/blob/7a3379b6878562e4958e61516335e912716588b7/service/kas/access/attribute.go#L10)  (as this is just a simplification of the existing policy struct for logging purposes)

I can change it no problem, but wondering if it's a wider change we'd want to make including the attribute struct as well in KAS?

Reply:
Awesome, I'm going to resolve this conversation then.